typescript/packages/plugins/balmy/src/balmy.plugin.ts

```diff
@@ -0,0 +1,692 @@
+    },
+    ETHEREUM_GOERLI: {
+        chainId: 5,
+        name: "Ethereum Goerli",
+        ids: ["goerli"],
+        nativeCurrency: { symbol: "ETH", name: "Ethereum" },
+        wToken: "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+        publicRPCs: ["https://rpc.ankr.com/eth_goerli", "https://goerli.blockpi.network/v1/rpc/public"],
+        explorer: "https://goerli.etherscan.io/",
+        testnet: true,
+    },
+    ETHEREUM_SEPOLIA: {
+        chainId: 11155111,
+        name: "Ethereum Sepolia",
+        ids: ["sepolia"],
+        nativeCurrency: { symbol: "ETH", name: "Ethereum" },
+        wToken: "0xf531b8f309be94191af87605cfbf600d71c2cfe0",
+        publicRPCs: ["https://rpc.sepolia.org"],
+        explorer: "https://sepolia.etherscan.io/",
+        testnet: true,
+    },
+    POLYGON_MUMBAI: {
+        chainId: 80001,
+        name: "Polygon Mumbai",
+        ids: ["mumbai"],
+        nativeCurrency: { symbol: "POL", name: "Polygon Ecosystem Token" },
+        wToken: "0x9c3c9283d3e44854697cd22d3faa240cfb032889",
+        publicRPCs: [
+            "https://rpc.ankr.com/polygon_mumbai",
+            "https://polygon-mumbai.blockpi.network/v1/rpc/public",
+            "https://polygon-mumbai-pokt.nodies.app",
+        ],
+        explorer: "https://mumbai.polygonscan.com/",
+        testnet: true,
+    },
+    BASE_GOERLI: {
+        chainId: 84531,
+        name: "Base Goerli",
+        ids: ["base-goerli"],
+        nativeCurrency: { symbol: "ETH", name: "Ethereum" },
+        wToken: "0x4200000000000000000000000000000000000006",
+        publicRPCs: [
+            "https://goerli.base.org",
+            "https://base-goerli.public.blastapi.io",
+            "https://base-goerli.blockpi.network/v1/rpc/public",
+            "https://endpoints.omniatech.io/v1/base/goerli/public",
+            "https://base-goerli.gateway.tenderly.co",
+        ],
+        explorer: "https://goerli.basescan.org/",
+        testnet: true,
+    },
+};
+
+const SUPPORTED_CHAIN_IDS = [
+    1, // Ethereum
+    10, // Optimism
+    42161, // Arbitrum
+    42170, // Arbitrum Nova
+    137, // Polygon
+    56, // BNB Chain
+    8453, // Base
+    250, // Fantom
+    42220, // Celo
+    1088, // Metis
+    43114, // Avalanche
+    128, // Heco
+    66, // OKC
+    1285, // Moonriver
+    1284, // Moonbeam
+    122, // Fuse
+    106, // Velas
+    100, // Gnosis
+    25, // Cronos
+    288, // Boba
+    58, // Ontology
+    8217, // Kaia
+    1313161554, // Aurora
+    592, // Astar
+    1666600000, // Harmony
+    199, // BitTorrent
+    42262, // Oasis
+    204, // opBNB
+    7700, // Canto
+    9001, // Evmos
+    30, // Rootstock
+    1101, // Polygon zkEVM
+    2222, // Kava
+    59144, // Linea
+    34443, // Mode
+    81457, // Blast
+    534352, // Scroll
+    5000, // Mantle
+    // Testnets
+    5, // Goerli
+    11155111, // Sepolia
+    80001, // Mumbai
+    84531, // Base Goerli
+];
+
+const DEFAULT_CONFIG = {
```

Comment:
All this config should not be needed at all. You are simply passing default values which the `balmySDK` already sets. You should be able to call `buildSDK()`. Even all the network information is def not needed here, the SDK already defaults to public RPCs which is what you are doing.